source.yml

```diff
@@ -155,12 +155,10 @@ version_control:
             - $AUTOPROJ_SOURCE_DIR/patches/hogman_gcc_4.8.2.patch
     
     - slam/g2o:
-        type: svn
-        url: http://svn.openslam.org/data/svn/g2o/trunk
+        github: RainerKuemmerle/g2o.git
```

Comment:
You will have to warn the mailing list in advance for this, autobuild does not handle changing importers gracefully.